test/common/stream_info/stream_info_impl_test.cc

```diff
@@ -37,8 +37,10 @@ std::chrono::nanoseconds checkDuration(std::chrono::nanoseconds last,
 class StreamInfoImplTest : public testing::Test {
 protected:
   void assertStreamInfoSize(StreamInfoImpl stream_info) {
-    ASSERT_TRUE(sizeof(stream_info) == 824 || sizeof(stream_info) == 840 ||
-                sizeof(stream_info) == 872)
+    ASSERT_TRUE(sizeof(stream_info) == 800 || sizeof(stream_info) == 816 ||
```

Comment:
Rather than just adding checks here, I would try to see if we can keep this number of checks to 3 as before but fix them if needed.